src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -518,6 +518,9 @@ std::vector<std::string> disabledTestPatterns() {
         // tests are useless on such platforms
         retVector.emplace_back(R"(.*(BF|bf)16.*)");
         retVector.emplace_back(R"(.*bfloat16.*)");
+        // Issue: MFDNN-12876
```

Comment:
I remembered the case only fails on avx2_vnni platform? If so, skip the test cases only on the avx2_vnni?

Reply:
still failed on avx2 without vnni. So, disable these tests on non-avx512 platforms.